notebooks/official/tensorboard/tensorboard_vertex_ai_hyperparameter_tuning.ipynb

```diff
@@ -0,0 +1,807 @@
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "goitVEQmnz2u"
+      },
+      "source": [
+        "If you run into ImportError: cannot import name 'WKBWriter' from 'shapely.geos', try the following and then restart runtime:"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "init_aip:mbsdk,all"
+      },
+      "source": [
+        "### Initialize Vertex AI SDK for Python\n",
+        "\n",
+        "Initialize the Vertex AI SDK for Python for your project."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "KllitKlIu-GW"
+      },
+      "outputs": [],
+      "source": [
+        "aiplatform.init(project=PROJECT_ID, location=REGION)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Yj41fZkkfE0b"
+      },
+      "source": [
+        "Start by loading the TensorBoard notebook extension and importing TensorFlow and the TensorBoard HParams plugin:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "KSayPNqxfJC_"
+      },
+      "outputs": [],
+      "source": [
+        "# Load the TensorBoard notebook extension\n",
+        "%load_ext tensorboard\n",
+        "\n",
+        "# Clear any logs from previous runs\n",
+        "!rm -rf ./logs/\n",
+        "\n",
+        "# Import TensorFlow and the TensorBoard HParams plugin\n",
+        "import tensorflow as tf\n",
+        "from tensorboard.plugins.hparams import api as hp"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "KJ4zE7rYfcvb"
+      },
+      "source": [
+        "Download the [FashionMNIST](https://github.com/zalandoresearch/fashion-mnist) dataset and scale it."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "vHME9wnnfiMr"
+      },
+      "outputs": [],
+      "source": [
+        "fashion_mnist = tf.keras.datasets.fashion_mnist\n",
+        "\n",
+        "(x_train, y_train), (x_test, y_test) = fashion_mnist.load_data()\n",
+        "x_train, x_test = x_train / 255.0, x_test / 255.0"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ofGSMru5r4kP"
+      },
+      "source": [
+        "## Experiment setup and the HParams experiment summary\n",
+        "\n",
+        "Experiment with three hyperparameters in the model:\n",
+        "\n",
+        "1. Number of units in the first dense layer\n",
+        "2. Dropout rate in the dropout layer\n",
+        "3. Optimizer\n",
+        "\n",
+        "List the values to try, and log an experiment configuration to TensorBoard. This step is optional: you can provide domain information to enable more precise filtering of hyperparameters in the UI, and you can specify which metrics should be displayed."
```

Comment:
Replace with:

Specify the hyperparameter values for the experiment in TensorBoard.

**Optional*: For more fine grained filtering of hyperparameters in the UI, provide domain information and specify which metrics should be displayed.